### ISSUE TITLE
A11y_Visual Studio Code Web Extensions_Github Pull request_Create Pull request_Keyboard: Focus is not visible on combo box in 'Merge changes from' section while navigating with tab key.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/vscode-pull-request-github#4244]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#GH_Visual Studio Code Web Extensions_Nov2022;#Visual Studio Code Web Extensions ;#ChromiumEdge;#DesktopWeb;#A11ySev2;#Win11;#WCAG2.4.7;#Keyboard;#Closed;

## Environment Details:
Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Windows Version: Win11
Edge version:107.0.1418.56

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Open https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
3. tab to 'Github' and press enter.
4. tab to 'Github Pull request' and press enter.
5. Check using tab key focus is visible on combo box in 'Merge changes from' section or not.

## Actual:
Focus is not visible on combo box in 'Merge changes from' section while navigating with tab key.
**Similar issue observed in 'INTO' Section on same screen.**

## Expected:
Focus should be visible on combo box in 'Merge changes from' section while navigating with tab key.

## User Impact:
Keyboard user will not know where focus has landed.

## Attachment:
![FocusNotVisible](https://user-images.githubusercontent.com/85863999/203893191-c8e82c6b-9d67-434a-aade-7d8720fc2944.gif)

**Comment by sspatil123** [comment]
The issue is fixed in below environment. Hence closing bug. https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
![4244_Fixed](https://user-images.githubusercontent.com/85863999/209780909-54416200-3716-4253-8726-b2b9e6cdc1dc.gif)
